File: playground/BOJ/boj_10818/boj_10818.py
'''
----------------------------------------------
Sub : [BOJ] 최소, 최대
Link: https://www.acmicpc.net/problem/10818
Tag : Python, math
Memo
----------------------------------------------
'''

n = int(input())  # 테스트 케이스 입력
nums = list(map(int, input().split()))
print(min(nums), max(nums)) # 최소, 최대 출력

File: playground/BOJ/boj_10818/boj_10818.js
/*
----------------------------------------------
Sub : [BOJ] 최소, 최대
Link: https://www.acmicpc.net/problem/10818
Tag : JS, math
Memo
----------------------------------------------
*/

const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const arr = input[1].split(' ').map(Number);

console.log(Math.min(...arr) + ' ' + Math.max(...arr));
